## Reference genome updates benefit DNA methyaltion study
[![Github](https://img.shields.io/badge/-Github-000?style=flat&logo=Github&logoColor=white)](https://github.com/functionalepigenomics)
[![Gmail](https://img.shields.io/badge/-Gmail-c14438?style=flat&logo=Gmail&logoColor=white)](mailto:dzheng.th@gmail.com)

This repository contains codes codes to reproduce the results and figures published in the paper.

"Complete Reference Genome and Pangenome Expand Biologically Relevant Information for Genome-Wide DNA Methylation Analysis Using Short-Read Sequencing and Array Data"

The unambiguous probe sets on the HM450K, EPIC, and EPICv2 arrays assessed by T2T-CHM13, pangenome, and their combination (95 assembles including T2T-CHM13 and 94 assembles from the pangenome), separately, are available for download from https://github.com/functionalepigenomics/Illumina_Infinium_HumanMethylation_BeadChips_Annotation.

### :computer: Repository Structure
The repository is organized as follows:

- Figures: This folder contains codes for generating the main figures.

- Short-read sequencing: This folder contains scripts related to analyzing DNA methyaltion in short-read sequencing data.

- data: This folder contains some input data for running the code in the notebooks. Due to the store limitation, some of the large data files are not included in the repository. However, instructions are provided on how to obtain the required data.

Unambigous_probes_identification: This folder stores codes for the identification of unambiguous probe sets in DNA methyaltion arrays.

Please note that some of the codes can be easily modified to perform similar analyses, as with the HM450K, EPIC, and EPICv2 arrays, and only one copy is provided here for easier follow-up.

### ðŸ“« Contact me
Email: dzheng.th@gmail.com
